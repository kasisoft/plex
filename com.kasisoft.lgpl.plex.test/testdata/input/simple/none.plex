<?xml version="1.0" encoding="UTF-8"?>
<plex>

  <general>
    <interface api="transform" id="cleanup"          classname="com.kasisoft.lgpl.plex.impl.CleanupTransform"     />
  </general>

  <sheet name="none" firstrow="1">
    <column title="key"       column="1000">
      <transformer refid="cleanup"      />
    </column>
    <column title="value"     column="1001">
      <transformer refid="cleanup"      />
    </column>
  </sheet>
  
</plex>