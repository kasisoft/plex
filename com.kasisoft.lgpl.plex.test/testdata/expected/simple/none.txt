<workbook>
  <sheet name="none" columns="2" rows="0">
    <column name="key" type="java.lang.Object"/>
    <column name="value" type="java.lang.Object"/>
  </sheet>
</workbook>
